jsapp/js/components/ccpmReportContent.es6

```diff
@@ -172,6 +172,32 @@ export default class CCPM_ReportContents extends React.Component {
     componentDidMount(){
       this.setReportData(this.props.reportData);
     }
+
+    getP12Question() {
+      const {parentState: {p12Result}} = this.props;
+      const no = p12Result.filter(res => res['Partner_Survey_GROUP/Partner_Inform_Strategy_GROUP/P_IS02'] === 'no');
```

Comment:
Do you think there is a way to identify P_IS02 without specifying the whole question path? Just wondering in case future surveys are named or grouped differently, so the question finding will fail.